src/components/ui/Tag.js

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components'
+
+const UnstyledTagList = ({ children, className }) => (
+  <ul className={className}>{children}</ul>
+)
+
+const TagList = styled(UnstyledTagList)`
+  padding: 0;
+  list-style: none;
+`
```

Comment:
Here it seems better to style with

```
const TagList = styled.ul`
  padding: o;
  list-style: none;
`
```

You don't need the Unstyled intermediate tag.